package.json

```diff
@@ -86,9 +86,9 @@
     "prettier": "^1.19.1",
     "progress-estimator": "^0.2.2",
     "regenerator-runtime": "^0.13.7",
-    "rollup": "^1.32.1",
+    "rollup": "^2.28.2",
     "rollup-plugin-sourcemaps": "^0.6.2",
-    "rollup-plugin-terser": "^5.1.2",
+    "rollup-plugin-terser": "^7.0.2",
```

Comment:
[v6.0.0](https://github.com/TrySound/rollup-plugin-terser/releases/tag/v6.0.0) requires Rollup 2 and Node 10+. c.f. https://github.com/formium/tsdx/pull/731#issuecomment-672463413